roads.mss

```diff
@@ -393,6 +394,25 @@ as well. */
 }
 
 /* ================================================================== */
+/* FERRIES
+/* ================================================================== */
+
+#ferry {
+  [route='ferry'] {
```

Comment:
Same remark here: no zoom in the selector means all zoom, so better to add zoom filter in the selector itself.